AcadeMe/views.py

```diff
@@ -12,3 +11,11 @@ def homePage(request):
 
 def contact(request):
     return render(request, 'landing/contact_us.html')
+
+
+def loginPage(request):
+    return render(request, 'login/login.html')
+
+
+def registrationPage(request):
+    return render(request, 'registration/registration.html')
```

Comment:
Just regular render methods for login and registration html files
used to render these files to the screen just like the ones above them